docs/user_guide/source/setting_up/package.rst

```diff
@@ -6,23 +6,37 @@ As Package
 Conda
 *****
 
-Currently ADIOS 2 can be obtained from anaconda cloud:
+The ADIOS 2 Python bindings can be obtained from anaconda cloud:
```

Comment:
other bindings are also available

Reply:
Via anaconda?

Reply:
Yes, C, C++, and Fortran are included. In fact, adios2 C++ jupyter notebooks on [binder](https://mybinder.org/v2/gh/ornladios/ADIOS2-Jupyter.git/cxx11) require conda and xling as backends.